requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 -r requirements.txt
 
+mypy
```

Comment:
This should have a version range like other dependencies.

Reply:
`pytest*` and `pylint-pydantic` don't have versions set btw. (Maybe they should.)

In general, how do I pick the minimal version? For now, I will pick the latest stable (`>=1.10.0`) and cap it with `<2.0`. But if it's wrong, please let me know.